src/frontend/apps/standalone_site/src/features/Playlist/components/CreatePlaylistForm.tsx

```diff
@@ -96,6 +96,23 @@ export const CreatePlaylistForm = () => {
       history.push(routes.PLAYLIST.path);
     },
   });
+  const [isInitOrganization, setIsInitOrganization] = useState(false);
+
+  useEffect(() => {
+    if (isInitOrganization) {
+      return;
+    }
+
+    if (!organizationResponse || organizationResponse.count === 0) {
+      return;
+    }
+
+    setFormValues((currentValue) => ({
+      ...currentValue,
+      organizationId: organizationResponse.results[0].id,
+    }));
+    setIsInitOrganization(true);
```

Comment:
can not be made in the `onSuccess` of `useOrganizations` ?

Reply:
It definitely can, would you prefer ?
I don't see any difference

Reply:
less hook and the `onSuccess` is made for this ?